charts/tfy-gpu-operator/values.yaml

```diff
@@ -20,6 +20,12 @@ clusterType:
 ## @section aws-eks-gpu-operator Configuration for the AWS EKS GPU Operator. This section will only be used when clusterType.awsEks is set to true.
 ##
 aws-eks-gpu-operator:
+  nfd:
+    enabled: true
+  
+  gfd:
+    enabled: true
+
   ## Driver configuration
   ## @param aws-eks-gpu-operator.driver.enabled Enabled/Disable driver installation.
   driver:
```

Comment:
This disabling needs testing actually, might cause the toolkit pods to be in pending